packages/virtualenv.yaml

```diff
@@ -0,0 +1,8 @@
+description: "virtualenv"
+webPage: https://virtualenv.pypa.io/en/stable//
```

Comment:
I'd drop everything after `.io/`. That will redirect to `/en/stable/`, is less likely to change over time, and in theory allows folks in other locales to use kyrios.

Reply:
Makes sense to me, I'll change that.